config/default.js

```diff
@@ -109,28 +109,39 @@ module.exports = {
     },
     support: {
         category: '740654285154549901',
-        automaticDeletion: 3600000,
         types: {
             'minez': {
                 description: "Support for MineZ: Stuck in a block; Broken dungeons; Death due to cheaters, bugs, and broken dungeons.",
-                roles: ['618856455206076417']
+                roles: [{ id: '618856455206076417', ping: true }],
+                logRoom: '748963030888349736',
+                autoClose: {
+                    warning: ONE_DAY_IN_MS,
+                    closing: 2 * ONE_DAY_IN_MS
+                }
             },
             'abuse': {
                 description: "Open a private chat with the leadership team to report staff abuse.",
-                roles: ['440505209580683274', '441611057878794240'],
+                roles: [
+                    { id: '440505209580683274', ping: true },
+                    { id: '441611057878794240', ping: true }
+                ],
+                logRoom: '748963030888349736',
+                autoClose: {
+                    warning: ONE_DAY_IN_MS,
+                    closing: 2 * ONE_DAY_IN_MS
+                }
+            },
+            'bug': {
+                description: "Open a support channel to report a bug to the development team.",
+                roles: [{id: '223821346763833344', ping: true}],
```

Comment:
Can you please change this? `223821346763833344` (Admin) should be included but not pinged. `815871997871128576` (Buganeer) should be included and pinged.

Reply:
Done